packages/vaadin-lumo-styles/src/components/checkbox.css

```diff
@@ -242,7 +242,7 @@
 }
 
 :host([invalid]) [part='checkbox'] {
-  background: var(--_invalid-background);
+  background-color: var(--_invalid-background);
```

Comment:
Let's also update the `.js` file accordingly.

Reply:
Done.